framework/include/interfaces/BoundaryRestrictable.h

```diff
@@ -220,12 +233,59 @@ class BoundaryRestrictable
    */
   void initializeBoundaryRestrictable();
 
+#ifdef MOOSE_KOKKOS_ENABLED
+  void initializeKokkosBoundaryRestrictable(MooseMesh * mesh);
+#endif
+
 protected:
   /**
    * A helper method to avoid circular #include problems.
    * @see hasBoundaryMaterialProperty
    */
   bool hasBoundaryMaterialPropertyHelper(const std::string & prop_name) const;
+
+#ifdef MOOSE_KOKKOS_SCOPE
```

Comment:
I thought I commented on this already, but maybe not. These methods are only used for kokkos - should they have a name that describes them as such?